Dockerfile

```
@@ -1,5 +1,11 @@
 FROM camptocamp/python-action
 
+RUN \
+  apt-get update && \
+  apt-get install --assume-yes --no-install-recommends git && \
+  apt-get clean && \
+  rm --recursive --force /var/lib/apt/lists/
```

You probably temporarily need to change the user, with something like that:
```suggestion
USER root

RUN \
  apt-get update && \
  apt-get install --assume-yes --no-install-recommends git && \
  apt-get clean && \
  rm --recursive --force /var/lib/apt/lists/
  
USER 1001:115
```